src/player.cpp

```diff
@@ -12942,6 +12976,23 @@ int player::get_hp_max( hp_part bp ) const
     return hp_total;
 }
 
+int player::get_stamina_max()
+{
+    return 1000;
```

Comment:
OK.  I was thinking more along the lines of being able to swing that melee weapon around more, not necessarily running, but that means touching the process in a different area.  Thanks.